pkg/auth/providers/publicapi/login.go

```diff
@@ -208,9 +213,61 @@ func (h *loginHandler) createLoginToken(request *types.APIContext) (v3.Token, st
 		if err != nil {
 			return v3.Token{}, "", "", err
 		}
+
+		if err := h.createClusterAuthTokenIfNeeded(token, tokenValue); err != nil {
+			logrus.Infof("Failed to create cluster auth token for cluster [%s], cluster auth endpoint may fail: %v", token.ClusterName, err)
```

Comment:
don't we want to return error too? for cli through kubectl if this actually goes through, when would users know the issue? 
Especially since we're not retrying, this might happen often.

Reply:
The problem is that the regular token was created, so unless we clean it up here a 201 is accurate.

Reply:
Thats tough problem. Shouldn't we jitter the create below to ensure thats extreme edge case ? Otherwise we should discuss what to do here.